terraform-modules/stackdriver/uptime-check/variables.tf

```diff
@@ -30,11 +30,76 @@ variable "notification_channels" {
   default     = []
 }
 
+variable "alert_labels" {
+  type        = map(string)
+  description = "Extra user labels to apply to the alert policy"
+  default     = {}
+}
+
 locals {
   check_name = "${var.service}-uptime"
   port       = var.https_enabled ? "443" : "80"
 }
 
+variable "enable_revere_service_labels" {
+  type        = bool
+  default     = false
+  description = <<-EOT
+    Whether to enable Revere's labeling scheme that allows it to
+    programmatically understand notifications triggered by this
+    alert policy.
+
+    More information on Revere's label usage available
+    [here](https://github.com/broadinstitute/revere/blob/main/docs/gcp_alert_policy_labels.md).
+  EOT
+}
+
+variable "revere_service_name" {
+  type        = string
+  default     = null
+  description = <<-EOT
+    Override the name of the service this alert monitors as understood by
+    [Revere](https://github.com/broadinstitute/revere).
+
+    By default, uses all but the last part of `var.service` (splitting on hyphens).
+  EOT
+}
+
+variable "revere_service_environment" {
+  type        = string
+  default     = null
+  description = <<-EOT
+    Override the environment of the service this alert monitors as understood by
+    [Revere](https://github.com/broadinstitute/revere).
+
+    By default, uses the last part of `var.service` (splitting on hyphens).
+  EOT
+}
+
+variable "revere_service_degredation" {
+  type        = string
+  default     = "uptime"
```

Comment:
hm, again a bit confused about other potential values here

Reply:
I think we'll probably change this based on discussion in https://github.com/broadinstitute/revere/pull/15